README.md

```diff
@@ -42,6 +42,12 @@ noble.on('discover', async (peripheral) => {
   await peripheral.disconnectAsync();
   process.exit(0);
 });
+```
+## Use Noble With BLE5 Extended Features With HCI 
+
+```javascript
+const noble = require('@abandonware/noble/with-custom-binding')({extended: false});
```

Comment:
should exteded be not true in this example?

Reply:
Only if we don't want to use ble5 features =) I will change it.

Reply:
;-) I think the example exactly "want" that ;-)) Thank you